htdocs/reports/reports_testhistory.php

```diff
@@ -2079,13 +2106,70 @@ function change_to_bold() {
 	}
 }
 ?>
-<div>
+<div id="signature">
 Signed by:
 <label>
 	<?php $user = get_user_by_id($signature);
 	echo $user->actualName; ?></label>
 </div>
 
+<div id="print_div" class="hidden"></div> <!--Content when the 1 test per page checkmark is checked is moved here-->
+
+<script>
+		const reportTable = "#patient_table";
+		const copiedElements = ["#logo", "#lab_header", "#report_header", "#patient_info_header"]
+		const printDiv = '#print_div';

```

Comment:
TIL, `const` [works in old versions of Firefox](https://caniuse.com/const) (like the bundled one) but it is the same as `var`. (something to keep in mind)

Reply:
Same thing as above, I removed the es6 features.